src/padding.rs

```diff
@@ -0,0 +1,53 @@
+use crate::CellOption;
+use papergrid::{Entity, Grid, Settings};
+
+const DEFAULT_PADDING_CHAR: char = ' ';
+
+/// Padding is responsible for a left/right/top/bottom indent of particular and fill one set characters.
+///
+/// ```rust,no_run
+///   # use tabled::{Style, Padding, Row, Table, Modify};
+///   # let data: Vec<&'static str> = Vec::new();
+///     let table = Table::new(&data).with(Modify::new(Row(..1)).with(Padding::new(0, 0, 1, 1).set_char('V', '^', '<', '>')));
+/// ```
+#[derive(Debug)]
+pub struct Padding(usize, char, usize, char, usize, char, usize, char);
+
+impl Padding {
+    /// Construct's an Indent object.
+    pub fn new(left: usize, right: usize, top: usize, bottom: usize) -> Self {
+        Self(
+            left,
+            DEFAULT_PADDING_CHAR,
+            right,
+            DEFAULT_PADDING_CHAR,
+            top, DEFAULT_PADDING_CHAR,
+            bottom,
+            DEFAULT_PADDING_CHAR)
+    }
+
+    pub fn set_char(mut self, left: char, right: char, top: char, bottom: char) -> Self {
```

Comment:
Do you think is there any better name for this function?
Or maybe it's better to set these characters separately?

Personally I am not sure.

Reply:
fixed